docs/references/errors.md

```diff
@@ -170,12 +191,33 @@ This is a generic error which can be caused by many reasons, including:
 
 We suggest checking the service and/or deployment logs as well to get any hint on the error cause.
 
-## RT0007
+## RT0007 {#RT0007}
 
 A retry-able error was received from the service deployment while processing the invocation.
 
 Suggestions:
 
 * Check the service/deployment logs to get more info about the error cause, like the stacktrace.
-* Look at the [error handling docs](/develop/sdk/error-handling) for more info about error handling in services.
+* Look at the https://docs.restate.dev/services/sdk/error-handling for more info about error handling in services.
```

Comment:
Replace by `[error handling docs](/develop/sdk/error-handling)`

Reply:
Will correct it in the Restate repo.